src/lesson5/ReverseTheText.java

```diff
@@ -0,0 +1,7 @@
+package lesson5;
+
+public class ReverseTheText {
+    public static String reverseTheText(String text){
+        return new StringBuilder(text).reverse().toString();
+    }
```

Comment:
no need to create a class with static method for call new StringBuilder(text).reverse().toString(); you can call it directly